WORKSPACE

```diff
@@ -56,3 +56,6 @@ pinned_maven_install()
 load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
+
+load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
+rules_pkg_dependencies()
```

Comment:
We might need to call this helper inside `protobuf_deps()` so that projects that depend on us also pick up the necessary dependencies.

Reply:
I think these must be present in WORKSPACE and can't be called in protobuf_deps because they are defined lazily. This is similar to skylib.